client/src/components/HomePage.tsx

```diff
@@ -11,13 +16,138 @@ const MainHeader = styled.h1`
   width: 100%;
 `;
 
-export type UserIdProps = {
-  userId: number;
-  setUserId: (userId: number) => void;
-};
+const SecondHeader = styled.h1`
+  font-size: 4rem;
+  color: ${CONTENT_COLOR};
+  text-align: center;
+`;
+
+const Content = styled.h2`
+  font-size: 3rem;
+  color: ${PRIMARY_COLOR};
+  text-align: center;
+`;
+
+const ContentText = styled.h3`
+  color: ${CONTENT_COLOR};
+  text-align: center;
+`;
+
+const ContentBlurb = styled.section`
+  border-bottom: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-top: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-right: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-left: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  flex-direction: row;
+  justify-content: space-between;
+  height: 220px;
+  width: 949px;
+  float: left;
+  background: ${PRIMARY_BACKGROUND_COLOR};
+`;
+
+const RedText = styled.h3`
+  color: ${"#A52A2A"};
+  text-align: center;
+`;
+
+const Background1 = styled.section`
+  background: ${PRIMARY_BACKGROUND_COLOR};
+  padding: 1em;
+  border-bottom: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  justify-content: space-between;
+`;
 
 const HomePageContent = () => (
-  <MainHeader>MOONLIGHT HEALTH</MainHeader>
+  <>
+    <MainHeader>MOONLIGHT HEALTH</MainHeader>
+
+    <Background1>
+      <RedText>COMPETITIVE ADVANTAGE FOR GP & ORTHODONTIC OFFICES</RedText>
+      <SecondHeader>
+        Put an Oral Surgeon <br></br>
+        In Your Practice
+      </SecondHeader>
+    </Background1>
+
+    <Content>Stop sending money out your door…</Content>
+    <ContentText>
+      Have you ever stopped to think how much revenue you are sending out the
+      door in oral surgery referrals? <br></br>
+      If your practice refers out even 5 patients a month, and that patient
+      receives an average of $3,000 worth <br></br>
+      of treatment, that is over $180,000 annually that the referring practice
+      doesn't see a dime of.
+    </ContentText>
+    <ContentText>
+      What if the surgeon worked for you? What if you received a portion of the
+      revenue generated by your valuable <br></br>
+      referrals, without having to worry about the liability, risk, and
+      equipment involved in hiring a surgical associate?
+    </ContentText>
+
+    <Content>Contact Info</Content>
+    <ContentText>Location: DC/MD/VA/FL</ContentText>
+    <ContentText>Email: dr.patel@moonlightoralsurgery.com</ContentText>
+    <ContentText>Phone: (408) 506-8919</ContentText>
+
+    <Content>Advantages</Content>
+    <ContentBlurb>
+      <RedText>More Convinient</RedText>
+      <ContentText>
+        Our services improve the general dentist’s practice by bringing oral
+        surgery <br></br>
```

Comment:
no need to do it this PR, we should look into using widths here instead of putting break lines on every line